views/new_peep.erb

```diff
@@ -0,0 +1,4 @@
+<form action="/peeps" method="post">
+  <input type="text" name="peep" placeholder="Enter peep" />
+  <input type="submit" value="Submit" />
+</form>
```

Comment:
for the next project you should try to implement more funcionalities on the home page, example, forms to take you to other pages